.github/workflows/benchmark-command.yml

```diff
@@ -28,7 +28,7 @@ on:
           - good
           - medium
           - dev
```

Comment:
What does this signify? I'm trying to align others with the appropriate presets while questioning if `dev` aligns with the same concept.

Reply:
Yup `dev` aligns with the same concept as well, these are all presets for different modules.
`dev` in particular is for timeseries, we can change the names but I have tried to keep this similar to the ones in AG-Bench repo to maintain uniformity.

You can find it in Bench here: https://github.com/autogluon/autogluon-bench/blob/master/sample_configs/amlb_configs/frameworks_example.yaml

and in the main repo here: https://github.com/autogluon/autogluon/blob/master/CI/bench/timeseries/amlb_user_dir/frameworks_template.yaml

It's listed as `_dev`

Reply:
ditto